### PR TITLE
docs(support-policy): add column for node 20 support

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
This goes in tandem w/ https://github.com/ionic-team/stencil/pull/4368

### CODE REVIEW

File: docs/reference/support-policy.md [hunk 1]
```diff
@@ -102,11 +102,11 @@ compatibility tables to describe the interoperability requirements of these piec
...
-| Stencil Version | Node v10 | Node v12 | Node v14 | Node v16 | Node v18  |  Deno*  |
-|:---------------:|:--------:|:--------:|:--------:|:--------:|:---------:|:-------:|
-|       V3        | &#10060; | &#10060; | &#9989;  | &#9989;  |  &#9989;  | &#9888; |
-|       V2        | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9888;** | &#9888; |
-|       V1        | &#9989;  | &#9989;  | &#9989;  | &#9989;  | &#10060;  | &#9888; |
+| Stencil Version | Node v10 | Node v12 | Node v14 | Node v16 | Node v18  |  Node v20 |  Deno*  |
+|:---------------:|:--------:|:--------:|:--------:|:--------:|:---------:|:---------:|:-------:|
+|       V3        | &#10060; | &#10060; | &#9989;  | &#9989;  |  &#9989;  | &#9989;   | &#9888; |
+|       V2        | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9888;** | &#10060;  | &#9888; |
+|       V1        | &#9989;  | &#9989;  | &#9989;  | &#9989;  | &#10060;  | &#10060;  | &#9888; |
```
Comment (alicewriteswrongs) [inline docs/reference/support-policy.md:109]: I figured we just treat v3 as one thing here, but should we make a separate row for v3.3 here instead?

**Review by alicewriteswrongs — COMMENTED** [review]
one question I had

File: docs/reference/support-policy.md (hunk 1 above)
Comment (rwaskiewicz) [reply docs/reference/support-policy.md:109]: When we added Node 18 support, that was slated for v3.1 - https://github.com/ionic-team/stencil-site/pull/955. I think leaving this at V3 for Node 20 is fine 👍

**Review by rwaskiewicz — APPROVED** [review]

File: docs/reference/support-policy.md (hunk 1 above)
Comment (tanner-reits) [reply docs/reference/support-policy.md:109]: When can we remove the Deno column?

**Review by tanner-reits — APPROVED** [review]

File: docs/reference/support-policy.md (hunk 1 above)
Comment (rwaskiewicz) [reply docs/reference/support-policy.md:109]: > When can we remove the Deno column?

I think we can remove that for/with the v4 release. I'll add a ticket to JIRA

Comment (rwaskiewicz) [reply docs/reference/support-policy.md:109]: STENCIL-835

**Pull request merged**